draft-irtf-cfrg-opaque.md

```diff
@@ -1586,7 +1593,7 @@ Expand-Label(Secret, Label, Context, Length) =
     Expand(Secret, CustomLabel, Length)
 ~~~
 
-Where CustomLabel is specified as:
+Where CustomLabel is specified and encoded (following Section 3.4 of {{?RFC8446}}) as:
```

Comment:
A reference to RFC8446 might potentially be overkill here.

You might also specify `CustomLabel` as the concatenation of `I2OSP(Length, 2)`, `I2OSP(len("OPAQUE-" + Label), 2)`, `"OPAQUE-"`, `Label`, `I2OSP(len(Context))`, and `Context`. This would also match how `Preamble` is specified in 6.4.2.1.

Reply:
Thanks! I contemplated doing it this way, but ultimately decided that referencing the section in RFC8446 was a more concise way to describe the concatenation. Hope that this is still clear enough.